deeppath/deeppath.py

```diff
@@ -38,6 +41,7 @@ def _get_repetition_index(key: str) -> Optional[Tuple[str, int]]:
     does not match the expected regex"""
     match = _REPETITION_REGEX.search(key)
     if match:
+        _logger.debug("key=%s, match groups=%s", key, match.groups())
```

Comment:
Actually can you avoid these debug lines or make sure they are suppressed unless specifically enabled? This is going to cause a lot (and I mean, a lot a lot) of logs...